File: python/ex050.py
# Desenvolva um programa que faça o sorteio de 20 números entre 0 e 10 e
# mostre na tela:
# a) Quais foram os números sorteados
# b) Quantos números estão acima de 5
# c) Quantos números são divisíveis por 3

import random

dividedBy3 = 0
aboveFive = 0
counter = 1

while counter <= 20:
    randNumbers = random.randrange(0,11) 
    if(randNumbers > 5):
        aboveFive+=1
        
    if(randNumbers % 3 == 0):
        dividedBy3+=1
    counter+=1
    print(f"Numeros sorteados: {randNumbers}")
print(f"Numeros acima de cinco: {aboveFive} \n Numeros divisiveis por 3: {dividedBy3}")

File: csharp/ex050.cs
// Desenvolva um programa que faça o sorteio de 20 números entre 0 e 10 e
// mostre na tela:
// a) Quais foram os números sorteados
// b) Quantos números estão acima de 5
// c) Quantos números são divisíveis por 3

class Ex050
{
  public int counter = 0;
  public int aboveFive = 0;
  public int divisibleThree = 0;

  public int randNumb = 0;

  Random randNum = new Random();

  public void FirstCountdown()
  {
    while (counter <= 20)
    {
      randNumb = randNum.Next(0, 10);
      Console.Write($"Numeros gerados : {randNumb}");

      if (randNumb > 5)
      {
        aboveFive += 1;
      }
      else if (randNumb % 5 == 0)
      {
        divisibleThree += 1;
      }
      
    }
    Console.Write($"Numeros acima de 5 : {aboveFive}");
    Console.Write($"Numeros divisiveis por 3 : {divisibleThree}");
  }
}

